src/main/scala/com/evolutiongaming/util/FutureEither.scala

```diff
@@ -144,6 +144,8 @@ object FutureEither {
     }.map(_.result())
   }
 
+  val unit: FutureEither[Nothing, Unit] = FutureEither(().ok)
```

Comment:
I would really go for method, I think that similar implementation in `Future` is wrong :)
```
private val _unit = FutureEither(().ok)

def unit[T]: FutureEither[T, Unit] = _unit
```